Review 190422 (#1)
uagpmc/handbook: docs/the-unit/reserves-system.md

### 3a. Requirements

> The only requirements of the reserves system is that you must be a full member, **shinies do not get to apply for reserves!**

Reviewer: ```suggestion
The only requirements of the reserves system is that you must be a full member, **shinies can not get to apply for reserves!**
```

Author: What about this? @Radrakin 

```suggestion
The only requirement of the reserves system is that you must be a full member, **shinies can't apply for reserves!**
```

Reviewer: Yep.

### 3b. Restrictions

The following restrictions are applied to all reserves who join a session:

> - Reserves choose their roles before shinies but after all other group members, and must fill less-desired roles if they are requested to by the section commander.

Reviewer 2: reserves choose their roles before shinies but after group members.
Conflict with "getting started" line 29 :
Shinies choose their roles before the rest of the group

Author: Resolved in cce7982d74405ef182997dcfc6cdaf04075bf94a

> - Reserves choose their roles before shinies but after all other group members, and must fill less-desired roles if they are requested to by the section commander.

Reviewer 3: How do Reserves pick before shinies but after the other members when shinies pick before the members

Author: Already resolved in cce7982d74405ef182997dcfc6cdaf04075bf94a
